magnum_capi_helm/tests/common/test_app_creds.py

```diff
@@ -67,6 +67,7 @@ def test_create_app_cred(self, mock_client):
                         "application_credential_secret": "pass",
                         "auth_url": "http://keystone",
                     },
+		    "auth_type" : "v3applicationcredential",
```

Comment:
I think the indentation isn't quite right here? The auto format should fix that if you just run "tox"